test/e2e/v4/generate_test.go

```diff
@@ -43,6 +43,7 @@ func GenerateV4(kbc *utils.TestContext) {
 		"--plugins", "go/v4-alpha",
 		"--project-version", "3",
 		"--domain", kbc.Domain,
+		"--component-config=true",
```

Comment:
Same could remove it or clear a mock test only for this?

Reply:
If I remove this flag, the test will fail because the` Kustomization.yaml` file is expecting the manager component config file to be generated.